frontend/src/components/Masthead/Search.tsx

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from "@material-ui/core/styles";
+import InputBase from "@material-ui/core/InputBase";
+import SearchIcon from "@material-ui/icons/Search";
+import { theme as appTheme } from "../../constants";
+
+const useStyles = makeStyles((theme) => ({
+  search: {
+    position: "relative",
+    background: appTheme.colors.inputBackground,
+    marginRight: theme.spacing(2),
+    width: "100%",
+    [theme.breakpoints.up("sm")]: {
+      marginLeft: theme.spacing(3),
+      width: "auto",
+    },
+  },
+  searchIcon: {
+    padding: theme.spacing(0, 2),
+    height: "100%",
+    position: "absolute",
+    pointerEvents: "none",
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  inputRoot: {
+    color: "inherit",
+  },
+  inputInput: {
+    padding: theme.spacing(1.3, 1, 1.3, 0),
+    paddingLeft: `calc(1em + ${theme.spacing(4)}px)`,
+    transition: theme.transitions.create("width"),
+    width: "100%",
+    [theme.breakpoints.up("md")]: {
+      width: "40ch",
+    },
+  },
+}));
+
+export function Search() {
+  const classes = useStyles();
+  return (
+    <div className={classes.search}>
+      <div className={classes.searchIcon}>
+        <SearchIcon />
+      </div>
+      <InputBase
```

Comment:
Couldn't we use our Input.tsx component here? 🤔

Reply:
Probably yes, I was to lazy to check